src/containers/ClassifierContainer.jsx

```diff
@@ -49,8 +50,9 @@ class ClassifierContainer extends React.Component {
     this.showCollections = this.showCollections.bind(this);
     this.showTutorial = this.showTutorial.bind(this);
     this.closePopup = this.closePopup.bind(this);
-    this.completeClassification = this.completeClassification.bind(this);
-    this.submitClassificationAndRedirect = this.submitClassificationAndRedirect.bind(this);
+    this.prepareSubmitClassificationForm = this.prepareSubmitClassificationForm.bind(this);
+    //this.completeClassification = this.completeClassification.bind(this);
+    //this.submitClassificationAndRedirect = this.submitClassificationAndRedirect.bind(this);
```

Comment:
Why are these merely commented out? Seems like their functionality is replicated in the submit form component.